sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client_models.hpp

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Defines the Azure Attestatoin API types.
+ *
+ */
+
+#pragma once
+
+#include "dll_import_export.hpp"
+
+#include "azure/core/context.hpp"
+#include "azure/core/http/http.hpp"
+#include "azure/core/nullable.hpp"
+#include "azure/core/paged_response.hpp"
+#include "azure/core/response.hpp"
+#include <iostream>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+// cspell: words MRSIGNER MRENCLAVE
+namespace Azure { namespace Security { namespace Attestation { namespace Models {
+  /**
+   * @brief Contains information about this instance of the attestation service, which can be used
+   * to validate attestation service responses.
+   *
+   */
+  struct AttestationOpenIdMetadata final
+  {
+    /// The issuer which will be used for tokens generated by this instance.
+    std::string Issuer;
+
+    /// A URI which can be used to retrieve the AttestationSigner
+    /// objects returned by the attestation service.
+    std::string JsonWebKeySetUrl;
+
+    /// The response types that are supported by the service.
```

Comment:
comments not in proper format 
/**
     * @brief Define policies to be called one time for every HTTP request from an SDK client.
     *
     */